src/Microsoft.FileFormats/ELF/ELFCoreFile.cs

```diff
@@ -30,6 +28,10 @@ public bool IsValid()
             return _elf.IsValid() && _elf.Header.Type == ELFHeaderType.Core;
         }
 
+        public bool Is64Bit => _elf.Is64Bit;
```

Comment:
Are these properties used?

Reply:
They will be.  I needed for this little dump tool that displays info about ELF dumps.